MEP_Engine/Query/SolidVolume.cs

```diff
@@ -0,0 +1,178 @@
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+using System.Collections.Generic;
+using System;
+
+using BH.oM.Reflection;
+using BH.oM.Reflection.Attributes;
+using BH.oM.MEP.Elements;
+using BH.Engine.Spatial;
+
+namespace BH.Engine.MEP
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Queries the solid volume of a Duct by multiplying the section profile's solid area by the element's length.")]
+        [Input("duct", "The Duct to query solid volume.")]
+        [MultiOutput(0, "elementSolidVolume", "SolidVolume of the Element itself within a compiled SectionProfile.")]
+        [MultiOutput(1, "insulationSolidVolume", "The solid volume of the Duct's exterior insulation.")]
+        [MultiOutput(2, "liningSolidVolume", "The solid volume of the Duct's interior lining.")]
+        public static Output<double, double, double> SolidVolume(this Duct duct)
+        {
+            double length = duct.Length();
+            double elementSolidVolume = duct.SectionProperty.ElementSolidArea * length;
+            double insulationSolidVolume = duct.SectionProperty.InsulationSolidArea * length;
+            double liningSolidVolume = duct.SectionProperty.LiningSolidArea * length;
+
+            if (duct.SectionProperty == null)
+            {
+                Engine.Reflection.Compute.RecordError("No section property defined.");
+                return null;
+            }
+            //Negative LiningThickness Warning
```

Comment:
please separate `if` statements with blank lines (also in the lines below as well as other methods in this file)